lib/docs.js

```diff
@@ -22,17 +22,29 @@ docWarrior.prototype.get = function(opts, callback){
 
   var date = opts.params && opts.params.date ? opts.params.date : null;
   var docs = [];
+  var docsWithSubdirectories = [];
 
-  async.each(opts.docs, function(docType, callback) {
-    self._processDocument(docType, date, opts.params, function(err, content){
+  opts.docs = ['test_terms_and_conditions_EN', 'freeif_EN', 'bpg_EN']//remove when going live
+
+  self._processSubdirectories(opts.docs, function(err, alldirectories) {
```

Comment:
I would potentially like to write this as an async waterfall, but for now I'd say this implementation isn't bad enough to be labelled tech debt as it still works with just a few lines, but overall the file could be a bit neater. Maybe something I will look to do post go live 👍